polaris-react/src/components/Page/components/Header/Header.scss

```diff
@@ -86,43 +36,9 @@ $action-menu-rollup-computed-width: 40px;
       // stylelint-enable declaration-no-important
     }
   }
-
-  @media print {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none !important;
-  }
-}
-
-.AdditionalNavigationWrapper {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  display: flex;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  flex: 1 0 auto;
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  justify-content: flex-end;
-
-  @media print {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: none !important;
-  }
-}
-
-///
-/// Main content
-
-.MainContent {
-  .Header:not(.mobileView) & {
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    display: flex;
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    align-items: center;
-  }
 }
 
 .TitleActionMenuWrapper {
-  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-  flex: 1 1 auto;
-
   .Header:not(.mobileView) & {
```

Comment:
Can this be removed? Seems like it's handled by `Box` now?

Reply:
Nice catch! https://github.com/Shopify/polaris/pull/7929/commits/bc28da39c58cc8498e75b335fa90e99e70e6f3e6

Reply:
You might able able to remove the entire `TitleActionMenuWrapper` selector as I don't think its being used?